spec/models/email_settings_spec.rb

```diff
@@ -0,0 +1,102 @@
+require 'spec_helper'
+
+describe EmailSettings do
+  let(:conference) { create(:conference, short_title: 'goto', start_date: Date.new(2014, 05, 01), end_date: Date.new(2014, 05, 06)) }
```

Comment:
`:shortitle` is set by sequence in factory. I don't want it to change between tests cause I am using it in expected hash:
```Ruby
'registrationlink' => 'http://localhost:3000/conference/goto/register',
 'conference_splash_link' => 'http://localhost:3000/conference/goto',
 'schedule_link' => 'http://localhost:3000/conference/goto/schedule',
```
`:stard/end_date` is set by:
```Ruby
start_date { Date.today }
end_date { 6.days.from_now }
```
This one although being same in every test run, depends on time zone. Check out this [failing travis build](https://travis-ci.org/sonalkr132/osem/builds/118696738#L3055). It was 7 May on travis server when it was 6 May on my PC.